reddit_decider/__init__.py

```diff
@@ -227,48 +228,48 @@ def _get_dynamic_config_value(
 
         res = decider_func(dynamic_config_name, ctx)
         if res is None:
-            return
+            return default
         error = res.err()
         if error:
             logger.warning(f"Encountered error in decider.get_value(): {error}")
```

Comment:
I'd change `"... decider.get_value(): ..."` to `"... {decider_func.__name__}: ..."` to specify the DC the log is coming from.